processors.go

```diff
@@ -121,29 +184,30 @@ var UUIDMap = safeUUIDMap{
 // init initializes the ProcessorCatalog map for all processors. A processor must be listed here to be accessible.
 func init() {
 	ProcessorCatalog = map[string]ProcessorFunc{
-		"AlphaNumericScrambler": ProcessorAlphaNumericScrambler,
-		"EmptyJson":             ProcessorEmptyJson,
-		"FakeStreetAddress":     ProcessorAddress,
-		"FakeCity":              ProcessorCity,
-		"FakeLatitude":          ProcessorLatitude,
-		"FakeLongitude":         ProcessorLongitude,
-		"FakeCompanyName":       ProcessorCompanyName,
-		"FakeEmailAddress":      ProcessorEmailAddress,
-		"FakeFirstName":         ProcessorFirstName,
-		"FakeFullName":          ProcessorFullName,
-		"FakeIPv4":              ProcessorIPv4,
-		"FakeLastName":          ProcessorLastName,
-		"FakePhoneNumber":       ProcessorPhoneNumber,
-		"FakeState":             ProcessorState,
-		"FakeStateAbbrev":       ProcessorStateAbbrev,
-		"FakeUsername":          ProcessorUserName,
-		"FakeZip":               ProcessorZip,
-		"Identity":              ProcessorIdentity, // Default: Does not modify field
-		"RandomBoolean":         ProcessorRandomBoolean,
-		"RandomDate":            ProcessorRandomDate,
-		"RandomDigits":          ProcessorRandomDigits,
-		"RandomUUID":            ProcessorRandomUUID,
-		"ScrubString":           ProcessorScrubString,
+		"AlphaNumericScrambler":        ProcessorAlphaNumericScrambler,
+		"UniqueAlphaNumericScrambler":  ProcessorUniqueAlphaNumericScrambler,
```

Comment:
Just a nit, but is it ok if we keep this list separated alphabetically?

Reply:
Alpha-ordered this and updated the README as well.